internal/attestation/azure/snp/validator.go

```diff
@@ -311,6 +311,9 @@ func (a *azureInstanceInfo) validateAk(runtimeDataRaw []byte, reportData []byte,
 	return nil
 }
 
+// HCLAkValidator uses the Host Compatibility Layer, which is written by Azure,
+// and sits between Hypervisor and CVM OS as part of CVM, to validate an
+// attestation report.
```

Comment:
```suggestion
// HCLAkValidator validates an attestation key issued by the Host Compatibility Layer (HCL).
// The HCL is written by Azure, and sits between the Hypervisor and CVM OS.
// The HCL runs in the protected context of the CVM.
```
Shorter sentences maybe make it easier to read? :)